package com.revature.Project_0.util.exceptions;

public class DataSourceException extends RuntimeException {

    public DataSourceException(String message, Throwable cause) {
        super(message, cause);
    }

}
